sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -348,4 +377,61 @@ void populateDefaults(AbstractTelemetryBuilder builder, Resource resource) {
             ContextTagKeys.AI_INTERNAL_SDK_VERSION.toString(), VersionGenerator.getSdkVersion());
         ResourceParser.updateRoleNameAndInstance(builder, resource, configuration);
     }
+
+    /**
+     * Creates an {@link AutoConfiguredOpenTelemetrySdkBuilder} based on the options set in the builder.
+     *
+     * @return An instance of {@link AutoConfiguredOpenTelemetrySdkBuilder}.
+     */
+    public AutoConfiguredOpenTelemetrySdkBuilder getOpenTelemetrySdkBuilder() {
+        return AutoConfiguredOpenTelemetrySdk.builder()
+            .addPropertiesSupplier(() -> {
+                Map<String, String> props = new HashMap<>();
+                props.put("otel.traces.exporter", AzureMonitorExporterProviderKeys.EXPORTER_NAME);
+                props.put("otel.metrics.exporter", AzureMonitorExporterProviderKeys.EXPORTER_NAME);
+                props.put("otel.logs.exporter", AzureMonitorExporterProviderKeys.EXPORTER_NAME);
+                props.put(AzureMonitorExporterProviderKeys.INTERNAL_USING_BUILDER, "true");
```

Comment:
should we make this configurable? it's default to true for now. or add a todo for future improvement?

Reply:
make which part configurable? thx